installer/datafiles/base_omsagent.data

```diff
@@ -180,6 +180,20 @@ fi
 # The real conf directory has to be owned by omsagent to be readable by the omsagent daemon
 chown -R omsagent:omsagent ${{CONF_DIR}}
 
+# Should run before onboarding script(Postinstall_300) and enabling OMS service(%Postinstall_500)
+%Postinstall_290
+if [ -f ${{OMSHOME}}/ruby/lib/ruby/gems/2.2.0/cache/mysql2-0.4.4.gem ]; then
+# Cleans up static mysql2 ruby gem if it exists in omsagent ruby path
+  ${{OMSHOME}}/ruby/bin/gem uninstall mysql2 1> /dev/null 2> /dev/null
+fi
+echo "----- Installing MySQL ruby gem -----"
+if ! ${{OMSHOME}}/ruby/bin/gem install mysql2 2>/dev/null ; then
+echo "MySQL ruby gem was not installed. MySQL ruby plugin would be disabled."
```

Comment:
Suggest indenting these four spaces (in an 'if' block). Also indent 187 two more spaces, and indent 192-194 four spaces.